src/ray/gcs/gcs_server/actor_info_handler_impl.cc

```diff
@@ -18,6 +18,27 @@
 namespace ray {
 namespace rpc {
 
+void DefaultActorInfoHandler::HandleCreateActor(
+    const ray::rpc::CreateActorRequest &request, ray::rpc::CreateActorReply *reply,
+    ray::rpc::SendReplyCallback send_reply_callback) {
+  RAY_UNUSED(reply);
+  RAY_CHECK(request.task_spec().type() == TaskType::ACTOR_CREATION_TASK);
+  auto actor_id =
+      ActorID::FromBinary(request.task_spec().actor_creation_task_spec().actor_id());
```

Comment:
would be nice to also log where this request comes from .e.g. worker id and caller id, which should be helpful for debugging